src/components/header/Header.tsx

```diff
@@ -2,17 +2,25 @@ import style from './Header.module.sass';
 import logo from './logo.svg';
 import menu from './menu.svg';
 import search from './search.svg';
-
 /**
  * The header of the website.
  */
-export default function Header() {
+
+interface Props {
+  toggleSideNav: Function
+}
+
+export default function Header(props:Props) {
+  const handleOnClickMenu = () => {
```

Comment:
There isn't a need for this function declaration. Can be directly called from props.